### PR TITLE
DX: manage certificates with Makefile

### PR DESCRIPTION
**Pull request by kaihendry**
- refactor: fetch token programmatically
- refactor: generate all certificates via a makefile and remove .gitignore so we can see them

### CODE REVIEW

File: scripts/run.sh [hunk 1]
```diff
@@ -1,34 +1,9 @@
...
-  if command_exists docker-compose; then
-    DOCKER_COMPOSE_CMD="docker-compose"
-  else
-    echo "Error: Neither 'docker-compose' nor 'docker compose' is available. Please install Docker Compose."
-    exit 1
-  fi
-fi
-
-export MO_OPCP_TOKEN=$MO_OPCP_TOKEN; export CSO_OPCP_TOKEN=$CSO_OPCP_TOKEN;$DOCKER_COMPOSE_CMD up "${@:2}"
+export MO_OPCP_TOKEN=$MO_OPCP_TOKEN; export CSO_OPCP_TOKEN=$CSO_OPCP_TOKEN;docker-compose up "${@:2}" --build
```
Comment (subnova) [inline scripts/run.sh:9]: The code to detect whether or not "docker compose" was a command or whether we needed to fall back to the deprecated "docker-compose" command was deliberate. Furthermore, whether or not to perform a build should be optional.

File: README.md [hunk 2]
```diff
@@ -21,26 +21,22 @@ MaEVe is implemented in Go 1.20. Learn more about MaEVe and its existing compone
...
 To get the system up and running:
 
-1. Run the [./scripts/generate-tls-cert.sh](./scripts/generate-tls-cert.sh) script which will create a server
-certificate for the CSMS
-2. Run the [./scripts/get-ca-cert.sh](./scripts/get-ca-cert.sh) script with a token retrieved from 
-the [Hubject test environment](https://hubject.stoplight.io/docs/open-plugncharge/6bb8b3bc79c2e-authorization-token) 
-to retrieve the V2G root certificate and CPO Sub CA certificates - remember to put your token argument within quotes
-3. Run the [./scripts/run.sh](./scripts/run.sh) script with the same token to run all the required components - again, don't forget the quotes around the token
+1. `(cd config/certificates && make)`
+1. Run the [./scripts/run.sh](./scripts/run.sh) script with the same token to run all the required components - again, don't forget the quotes around the token
```
Comment (subnova) [inline README.md:31]: Markdown is often read as text. Whilst a markdown renderer will correctly number things - this won't be true when reading text.

File: scripts/run.sh (hunk 1 above)
Comment (kaihendry) [reply scripts/run.sh:9]: Ok, I can revert, just trying to tidy up. I find '--build' a better default when it comes to development, as what inevitably happens, is you make a change and docker compose {down,up} doesn't reflect that change.

**Comment by kaihendry** [comment]
I've made some changes, https://github.com/Yelp/detect-secrets has just way too many false positives.

[detect-private-key](https://github.com/pre-commit/pre-commit-hooks/blob/main/pre_commit_hooks/detect_private_key.py) seems like the sanest start. Else revert helm?

**Pull request closed without merging**